packages/styles/variables.css

```
@@ -20,6 +20,7 @@
   --accent-medium: #283640;
   --accent-light: #d4dde0;
   --accent-success: #a5db75;
+  --accent-accessible-success: #4fa630;
```

```suggestion
  --accent-success-alternate: #4fa630;
```

From conversation with @schne324